tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -4305,6 +4307,43 @@ describe('scalar card', () => {
           }),
         ]);
       }));
+
+      it('emits dataTableColumnToggled when onRemoveColumn is called', fakeAsync(() => {
+        store.overrideSelector(getSingleSelectionHeaders, [
+          {
+            type: ColumnHeaderType.RUN,
+            name: 'run',
+            displayName: 'Run',
+            enabled: true,
+          },
+          {
+            type: ColumnHeaderType.VALUE,
+            name: 'value',
+            displayName: 'Value',
+            enabled: false,
+          },
+        ]);
+        store.overrideSelector(getCardStateMap, {
+          card1: {
+            rangeSelectionOverride: CardFeatureOverride.OVERRIDE_AS_ENABLED,
+          },
+        });
```

Comment:
So in the data here, it seems to me that you provide single selection, but in range is enabled. Do you want to test single or range here?

Reply:
My mistake here - will add testing for both.